lib/models/items/item.dart

```diff
@@ -1,5 +1,15 @@
 import 'package:guildwars2_companion/models/items/item_details.dart';
 
+enum ItemSection {
```

Comment:
I think Enums in capital letters look a bit better. Like: EQUIPMENT, BANK etc.

Reply:
Updated to all caps.